package.json

```diff
@@ -8,7 +8,8 @@
     "ios": "expo start --ios",
     "web": "expo start --web",
     "eject": "expo eject",
-    "test": "jest"
+    "test": "jest",
+    "prepare": "husky install"
```

Comment:
What does this do? :-? Do we need to add an install command here since it implies it can be run more than once.

Reply:
This command was automatically added when husky was added to the project. I don't think we need to install command since husky helps to automatically run linting when we try to commit

Reply:
Oh awesome, sounds good. Thanks for explanation!